piperider_cli/cli.py

```diff
@@ -134,7 +134,7 @@ def diagnose(**kwargs):
 @click.option('--skip-dbt', is_flag=True, help='Skip running dbt.')
 @add_options(debug_option)
 def run(**kwargs):
-    'Profile data source, run assertions, and generate report(s).'
+    'Profile data source, run assertions, and generate report(s). By default, the raw results are saved in ".piperider/outputs" and the reports are saved in ".piperider/reports".'
```

Comment:
Is `.piperider/reports` a new folder? I haven't seen it yet 
![image](https://user-images.githubusercontent.com/11186480/174972123-d1c3ad0e-656f-45c6-9fd2-52b817744c39.png)

Reply:
![image](https://user-images.githubusercontent.com/11186480/174972302-52111cba-43c9-40b3-afd0-4188b09a2be2.png)
^ from generating reports

Reply:
uh, reports were saved in '.piperider/reports'.
let me check it's correct or a bug.

Reply:
Going forward reports are exported to .piperider/outputs, we have updated the Next docs accordingly. The live doc site is still for ver 0.2.0

Reply:
Fixed